### PR TITLE
[v1.x] ONNX support for _split_v2

### PR DESCRIPTION
**Pull request by waytrue17**
## Description ##
ONNX support and unittest for _split_v2

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py [hunk 1]
```diff
@@ -4852,3 +4854,65 @@ def convert_contrib_div_sqrt_dim(node, **kwargs):
     ]
 
     return nodes
+
+
+@mx_op.register('_contrib_div_sqrt_dim')
```
Comment (Zha0q1) [inline python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py:4859]: why this operator too?

File: tests/python-pytest/onnx/test_operators.py [hunk 2]
```diff
@@ -1858,3 +1858,19 @@ def rand_check(out):
     def rand_check_nd(out):
         return rand_check(out.asnumpy())
     op_export_test('sample_multinomial', M, [x], tmp_path, mx_map=rand_check_nd, onnx_map=rand_check)
+
+
+@pytest.mark.parametrize("dtype", ['float32', 'int32', 'int64'])
```
Comment (Zha0q1) [inline tests/python-pytest/onnx/test_operators.py:1863]: Is the customer use case covered by the unit tests?

File: python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py [hunk 3]
```diff
@@ -1732,3 +1731,70 @@ def convert_logsoftmax(node, **kwargs):
...
+            create_tensor([axis], name+'_axis', kwargs['initializer'])
+            nodes = [
+                make_node('Split', input_nodes, output_nodes_, axis=axis),
+            ]
+            for i in range(sections):
+                nodes += [
+                    make_node("Squeeze", [output_nodes_[i], name+'_axis'], [output_nodes[i]]),
+                ]
+    else:
+        # indices += [0]
```
Comment (Zha0q1) [inline python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py:1765]: consider removing

**Review by Zha0q1 — COMMENTED** [review]
Thanks for the quick implementation! left a few minor comments

File: python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py (hunk 1 above)
Comment (waytrue17) [reply python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py]: Unintentionally pasted it. Removed.

File: python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py (hunk 3 above)
Comment (waytrue17) [reply python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py]: Done

File: tests/python-pytest/onnx/test_operators.py (hunk 2 above)
Comment (waytrue17) [reply tests/python-pytest/onnx/test_operators.py:1863]: Customer has not shared their use case yet

**Review by josephevans — APPROVED** [review]
LGTM, thanks!
